packages/fast-stable-stringify/src/index.ts

```diff
@@ -61,7 +61,7 @@ function stringify(val: unknown, isArrayProp: boolean) {
         case 'undefined':
             return isArrayProp ? null : undefined;
         case 'bigint':
-            return JSON.stringify(val.toString() + 'n');
+            return JSON.stringify(val.toString()) + 'n';
```

Comment:
Looks great! Can you think of a reason why we still need the `"` character though? Can we just do ```${val.toString()}n``` and have it work?

Reply:
Fixed it.